aiidalab/app.py

```diff
@@ -648,6 +717,24 @@ def _default_detached(self):
     def _default_busy(self):  # pylint: disable=no-self-use
         return False
 
+    @traitlets.validate("version_to_install")
+    def _validate_version_to_install(self, proposal):
```

Comment:
There is a race condition for this traitlets. The `self.available_version` is updated in `refresh()` which is triggered by `_observe_version_to_install`. This makes unit tests fail.